roles/manage_cnp/README.md

```diff
@@ -0,0 +1,100 @@
+# manage_cnp
+
+This Ansible Galaxy Role contains task files that assist in the management of CloudNativePG and EDB Postgres for Kubernetes.
+
+**The ansible playbook must be executed under an account that has full
+privileges.**
+
+## Requirements
+
+The only dependencies required for this ansible galaxy role are:
+
+  1. AWS EKS CLI, Azure CLI, or Google Cloud CLI - Depends directly on the target cloud
+  2. Kubectl
+  3. Ansible
+  4. Python packages: openshift, pyyaml, kubernetes, pyhelm
+     Installed with commands:
+     `pip install openshift pyyaml kubernetes`
+     `pip install pyhelm`
+  5. `kubernetes.core.helm_repository` - Ansible Module - Required for Helm.
+  6. `community.kubernetes.helm` - Ansible Module - Required for Helm.
```

Comment:
The `community.kubernetes` collection is a part of the `kubernetes.core` collection, there is a `kubernetes.core.helm` module that would be consistent with the rest of the `kubernetes.core` requirement.

Reply:
removed because helm is not being utilized